src/renderer/components/RecentConnections.tsx

```diff
@@ -0,0 +1,61 @@
+import '../scss/RecentConnections.scss';
+import { Link } from 'react-router-dom';
+import { Button } from 'react-bootstrap';
+
+const RecentConnections = () => {
+  return (
+    <div className="RecentWrapper">
+      <h1 className="Header">Recent Connections</h1>
+      <div className="TDiv">
+        <table className="table">
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Type of Connection</th>
+              <th>Address</th>
+              <th>User</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <Link to="/Execute">
+                <button className="buttonSelect" type="button">
+                  <td className="LinkTD">Connection1</td>
+                </button>
+              </Link>
+              <td>PostgreSQL</td>
+              <td>123.234.32.4</td>
+              <td>User1</td>
+            </tr>
+            <tr>
+              <Link to="/Execute">
+                <button className="buttonSelect" type="button">
+                  <td className="LinkTD">Connection2</td>
+                </button>
+              </Link>
+              <td>MySQL</td>
+              <td>123.234.32.17</td>
+              <td>User2</td>
+            </tr>
+            <tr>
+              <Link to="/Execute">
+                <button className="buttonSelect" type="button">
+                  <td className="LinkTD">Connection3</td>
+                </button>
+              </Link>
+              <td>Oracle</td>
+              <td>456.789.18.13</td>
+              <td>User3</td>
+            </tr>
+          </tbody>
```

Comment:
This will need to be dynamically generated in the future. It will have to fetch a list (of unknown length) connections and display them.
One connection is of type `ConnectionModel` which can be found in `model.ts`.
For now you could just make an array of ConnectionModels and write the React code to generate one row from a single model, then in the future get this list dynamically.